packages/jest-circus/src/legacy_code_todo_rewrite/jest-adapter-init.js

```diff
@@ -88,7 +94,7 @@ const runAndTransformResultsToJestFormat = async ({
         status = 'passed';
         break;
       case 'skip':
-        status = 'skipped';
+        status = 'pending';
```

Comment:
How about we change jasmine to call them skipped and make this 'skipped' throughout Jest?